pkg/ebpf/tracee.go

```diff
@@ -71,6 +70,17 @@ type Config struct {
 	ContainersEnrich   bool
 }
 
+type BPFObjConfig struct {
```

Comment:
I propose we move these 3 into a `bpfplugin` package or something of the sorts because it's really a start of that.
Later we can have plugins using it to register themselves etc. but for now it would be a good place for the types.

Reply:
Does it make sense to have this as part of libbpfgo instead?

Reply:
I think having logic for plugins in their own package makes sense. However since we don't know what the interface looks like I don't want to split it up too early.